.test-defs/Testgrid.yaml

```
@@ -0,0 +1,34 @@
+# Copyright 2019 Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: TestDefinition
+metadata:
+  name: testgrid
+spec:
+  owner: DL_5C5BE3E2970B9F404D0E2F50@sap.com
+  recipientsOnFailure:
+  - DL_5C5BE3E2970B9F404D0E2F50@sap.com
+
+  description: Run kubernetes conformance tests and push result files (e2e.log and junit_01.xml) to testgrid repository.
+
+  activeDeadlineSeconds: 10800
+
+  config:
```

Please remove indent to be consistent with `recipientsOnFailure`
```suggestion
  config:
  - type: env
     name: PUBLISH_TO_TESTGRID
     value: "true"
```